netbout-web/src/test/java/com/netbout/rest/MarkdownTxtmarkTest.java

```diff
@@ -213,11 +206,8 @@ public void breaksSingleLine() throws Exception {
             ),
             Matchers.equalTo(
                 Joiner.on(MarkdownTxtmarkTest.EOL).join(
-                    "<p>line1 ",
-                    "line2 ",
-                    "<br/>",
-                    "line2</p>",
-                    "<p>line3</p>"
+                    "<p>line1<br  />", "line2<br  /></p>",
```

Comment:
@dmzaytsev it is so strange, why we want to have `br` with two spaces inside? Sorry if I'm out of context